### PR TITLE
Add typings for Relations

### LINKED ISSUE #989: Model.getRelations is missing on typings
Model.getRelations [getRelations()](http://vincit.github.io/objection.js/#getrelations) is not defined in typings.

### PR DESCRIPTION
**Pull request by rgtk**
Fixes #989

### CODE REVIEW

File: typings/objection/index.d.ts [hunk 1]
```diff
@@ -183,8 +183,23 @@ declare namespace Objection {
...
+    modelClass: Model;
+    props: string[];
+    cols: string[];
+  }
+
   interface Relation {
-    // TODO should this be something other than a tagging interface?
+    name: string;
+    ownerModelClass: Model;
+    relatedModelClass: Model;
```
Comment (koskimas) [inline typings/objection/index.d.ts]: There are not actually `Model` instances but `Model` constructors, so you should use `ModelClass<any>` here.

File: typings/objection/index.d.ts [hunk 2]
```diff
@@ -183,8 +183,23 @@ declare namespace Objection {
...
+    ownerProp: RelationProperty;
+    relatedProp: RelationProperty;
+    joinModelClass: Function;
```
Comment (koskimas) [inline typings/objection/index.d.ts]: This is also a `ModelClass<any>`

**Comment by koskimas** [comment]
Thanks for this! Couple of small comments, then I'm ready to merge this.

**Comment by rgtk** [comment]
Thank you for being constructive. Fixed.

**Pull request merged**